lightning/src/ln/channelmanager.rs

```diff
@@ -520,7 +520,7 @@ pub(crate) const MAX_LOCAL_BREAKDOWN_TIMEOUT: u16 = 2 * 6 * 24 * 7;
 /// ie the node we forwarded the payment on to should always have enough room to reliably time out
 /// the HTLC via a full update_fail_htlc/commitment_signed dance before we hit the
 /// CLTV_CLAIM_BUFFER point (we static assert that it's at least 3 blocks more).
-const CLTV_EXPIRY_DELTA: u16 = 6 * 12; //TODO?
+const CLTV_EXPIRY_DELTA: u16 = 6 * 6; //TODO?
```

Comment:
Won't this affect more than just the tests? If this was intentional, could you make a separate commit explaining the change?

Reply:
See #849.